ConsentViewController/Classes/MessageWebViewController.swift

```diff
@@ -55,11 +55,13 @@ class MessageWebViewController: GDPRMessageViewController, WKUIDelegate, WKNavig
     var isPMLoaded = false
     let timeout: TimeInterval
     var connectivityManager: Connectivity = ConnectivityManager()
+    let messageLanguage: MessageLanguage
 
-    init(propertyId: Int, pmId: String, consentUUID: GDPRUUID, timeout: TimeInterval) {
+    init(propertyId: Int, pmId: String, consentUUID: GDPRUUID, messageLanguage: MessageLanguage, timeout: TimeInterval) {
         self.propertyId = propertyId
         self.pmId = pmId
         self.consentUUID = consentUUID
+        self.messageLanguage = messageLanguage
         self.timeout = timeout
         super.init(nibName: nil, bundle: nil)
         self.queryItems = [URLQueryItem(name: "site_id", value: "\(propertyId)"), URLQueryItem(name: "consentUUID", value: consentUUID)]
```

Comment:
We can add the `messageLanguage` to the `queryItems` array.

Reply:
Yes, added into `queryItems` array